examples/panel/book-recommender/rag_book_recommender.py

```diff
@@ -0,0 +1,89 @@
+import os
+import json
+import pickle
+import pandas as pd
+
+from pathlib import Path
+from openai import OpenAI
+
+client = OpenAI()
+
+file_path = os.path.join('assets', 'goodreads.csv')
```

Comment:
there's no documentation on how to get this file, I'm assuming this was downloaded from kaggle, if so, mention it

Reply:
[fixed](https://github.com/ploomber/doc/pull/127/files#diff-396e0215830b7ad98dc178d3fd362e3e0a577ab3509a784b915a8242e092be98R9-R11)

Reply:
Is this mentioned in the readme? it'd better to document this in the readme than the code

Reply:
[added](https://github.com/ploomber/doc/pull/127/files#diff-3b853450dda7ca635422b6a6d2c988f70e93aea79c8f8878384d0fa175e013e0R13-R16)